styles.css

```diff
@@ -0,0 +1,84 @@
+body {
+    text-align: center;
+    font-family: fantasy;
+}
+
+.p {
+    text-align: center;
+}
+
+#numbers {
+    display: flex;
+    justify-content: center;
+}
+
+.card {
+    display: inline-block;
+    background: url(./img/471175_3756119374126_1958905707_o.jpg) no-repeat;
```

Comment:
can you name this picture properly?

Reply:
all pictures and picture names have been updated